user/pdfgrep/template.py

```diff
@@ -0,0 +1,13 @@
+pkgname = "pdfgrep"
+pkgver = "2.2.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = ["--prefix=/usr/"]
```

Comment:
```suggestion
```
This is no longer needed, as gnu_configure automatically sets `--prefix=/usr`

Reply:
Done!